next-react-query-tailwind/src/components/TabNavigation/TabNavigation.tsx

```diff
@@ -29,7 +29,9 @@ function TabNavigation({
 
   const isCurrentTab = (path?: string) => {
     const matchPath = path === '' ? basePath : `${basePath}/${path}`;
-    return path === '' ? pathname === basePath : pathname.includes(matchPath);
+    return path === ''
+      ? pathname === basePath || pathname.includes('tree')
+      : pathname.includes(matchPath);
```

Comment:
So this has exceeded my personal definition of what a ternary should contain. Can we refactor this to be:
```suggestion
   if (path === '') {
     return pathname === basePath || pathname.includes('tree');
   }
   return pathname.includes(matchPath);
```

I'm just very opposed to logic in a ternary clause.